### PR TITLE
MIG 358: CA certificate errors in MTC console

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-358

MTC 1.4

CP to 4.6 and 4.7.

### CODE REVIEW

**Comment by XinRedhat** [comment]
LGTM

**Review by lbarbeevargas — COMMENTED** [review]
Some suggestions, but otherwise LGTM. Can you confirm that there are no xrefs to the sections that you changed the `id` for? For example:

[id='migration-restic-errors_{context}'] was changed to [id='migration-error-messages_{context}']

File: modules/migration-error-messages.adoc [hunk 1]
```diff
@@ -3,13 +3,41 @@
...
 This section describes common error messages and how to resolve their underlying causes.
 
-[id='restic-timeout-error_{context}']
-== Restic timeout error
+[id='ca-certificate-error-in-console_{context}']
+== CA certificate error in the {mtc-short} console
+
+If a `CA certificate error` message is displayed the first time you try to access the {mtc-short} console, the likely cause is the use of self-signed CA certificates in one of the clusters.
+
+To resolve this issue, navigate to the `oauth-authorization-server` URL displayed in the error message and accept the certificate. To resolve this issue permanently, add the certificate to your web browser's trust store.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply modules/migration-error-messages.adoc:16]: I know about that rule and I think it's stupid, especially for longer strings. But I will make that change.

File: modules/migration-error-messages.adoc [hunk 2]
```diff
@@ -3,13 +3,41 @@
...
+If a `connection has timed out` message is displayed in the {mtc-short} console after you have accepted a self-signed certificate, the likely causes are the following:
+
+* Interrupted network access to the OAuth server
+* Interrupted network access to the {product-title} console
+* Proxy configuration that blocks access to the `oauth-authorization-server` URL. See link:https://access.redhat.com/solutions/5514491[MTC console inaccessible because of OAuth timeout error] for details.
+
+To determine the cause of the timeout, perform the following steps:
+
+. Navigate to the {mtc-short} console and inspect the elements with your browser's web inspector.
+. Check the `migration-ui` Pod log:
```
[the remark this answers is not in the document]
Comment (apinnick) [reply modules/migration-error-messages.adoc:32]: No one told me the guidelines had changed. I only knew they were being discussed.

**Comment by apinnick** [comment]
> Some suggestions, but otherwise LGTM. Can you confirm that there are no xrefs to the sections that you changed the `id` for? For example:
> 
> [id='migration-restic-errors_{context}'] was changed to [id='migration-error-messages_{context}']

@lbarbeevargas  I checked that

**Comment by lbarbeevargas** [comment]
Updates LGTM.

**Pull request merged**